src/main/java/com/ndgl/spotfinder/domain/auth/service/AuthService.java

```diff
@@ -1,30 +1,41 @@
 package com.ndgl.spotfinder.domain.auth.service;
 
+import java.util.Set;
+
+import org.springframework.data.redis.core.RedisTemplate;
 import org.springframework.stereotype.Service;
 
 import com.ndgl.spotfinder.global.exception.ErrorCode;
 import com.ndgl.spotfinder.global.security.jwt.TokenProvider;
 import com.ndgl.spotfinder.global.security.redis.entity.RefreshToken;
 import com.ndgl.spotfinder.global.security.redis.repository.RefreshTokenRepository;
 
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
 @Service
 public class AuthService {
 	private final TokenProvider tokenProvider;
-	private final RefreshTokenRepository refreshTokenRepository;
+	private final RedisTemplate<String, String> redisTemplate;
 
-	public AuthService(TokenProvider tokenProvider, RefreshTokenRepository refreshTokenRepository) {
+	public AuthService(TokenProvider tokenProvider,
+		RedisTemplate<String, String> redisTemplate) {
 		this.tokenProvider = tokenProvider;
-		this.refreshTokenRepository = refreshTokenRepository;
+		this.redisTemplate = redisTemplate;
 	}
 
 	public boolean tokenStatusCheck(String accessToken) {
 		return tokenProvider.validateToken(accessToken);
 	}
 
 	public String getRefreshTokenFromRedis(String userId) {
-		RefreshToken refreshToken = refreshTokenRepository.findById("refreshToken:" + userId)
-			.orElseThrow(ErrorCode.MISSING_REFRESH_TOKEN::throwServiceException);
+		String key = "refreshToken:" + userId;
+		Object refreshToken = redisTemplate.opsForHash().get(key, "token");
```

Comment:
만약 String 타입으로 저장하셨다면, 변환해주는 것이 더 좋아보입니다.
```suggestion
		String refreshToken = (String)redisTemplate.opsForHash().get(key, "token");
```